resources/views/components/menu/account.blade.php

```diff
@@ -75,8 +72,8 @@
         <x-dropdown-item :link="url('controlpanel.php')">Settings</x-dropdown-item>
         <div class="dropdown-divider"></div>
         {{--<x-form :action="route('logout')">--}}
-        <x-form :action="url('request/auth/logout.php')">
-            <button class="dropdown-item">{{ __('Sign Out') }}</button>
+        <x-form :action="route('logout')">
+            <button class="dropdown-item">{{ __('Sign out') }}</button>
```

Comment:
message/icon.blade.php doesn't appear in this diff, so I'm putting the comment here indirectly.

Now that we have a Messages icon at the top level, do we still need a Messages menu item in the Account menu? Alternatively, should we remove the Messages button from the Nav Bar, keep the Messages link in the Account menu, and list unread messages under Notifications (with the tickets/claim notifications)?

If we keep the navbar item for notifications, would it be reasonable to report the number of unread messages in the tooltip for the navbar button? "Messages (2 unread)"?

Reply:
I think adding "x unread messages" as a notification item is a neat idea.
It would cut down the amount of items listed there.

Reply:
<img width="523" alt="Screenshot 2023-09-05 at 20 49 57" src="https://github.com/RetroAchievements/RAWeb/assets/1280590/9c730dea-53ff-4171-a925-c1f22f51ccb6">

See https://github.com/RetroAchievements/RAWeb/pull/1831/commits/358fe16e415f39a6d6e9ca2434cf3dde4f2e5e96